nodejs/week2/warmup/index.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");
+const app = express();
+
+app.get("/", (request, response) => response.send("nodejs week2 homework"));
+
+// server response for addition of given two numbers
+// http://localhost:port/numbers/add?firstNumber=<number here>&secondNumber=<number here>
+app.get("/numbers/add", (request, response) => {
+    const query = request.query;
+    response.send(`Sum of given two numbers: ${query.firstNumber} and ${query.secondNumber} is = ${parseInt(query.firstNumber) + parseInt(query.secondNumber)}`);
+})
+
+// server response for multiplication of given two numbers
+// http://localhost:port/numbers/multiply?firstNumber=<number here>&secondNumber=<number here>
+app.get("/numbers/multiply", (request, response) => {
+    const query = request.query;
```

Comment:
The controller works well. 👍 
However, in this task of the homework, we should be using params instead of queries, like the following: 
`/numbers/multiply/<first number here>/<second number here>`

Reply:
ok thanks a lot, i will update that soon

Reply:
you mean : using request.params instead of request.query 
then i can do as below: 

app.get("/numbers/multiply/:firstNumber/:secondNumber", (request, response) => {
    const firstNumber = parseInt(request.params.firstNumber);
    const secondNumber = parseInt(request.params.secondNumber);
    response.send(`Multiplication of given two numbers: ${firstNumber} and ${secondNumber} is = ${firstNumber * secondNumber}`);
})